samples/servicecontrol/adapter-sqlserver/sample.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Using adapter to centralize monitoring of multi-instance SQL Server
+component: SCTransportAdapter
+reviewed: 2017-05-11
+related:
+ - servicecontrol
+ - servicecontrol/plugins
+---
+
+## Prerequisistes
+
+ 1. [Install ServiceControl](/servicecontrol/installation.md).
+ 1. Using the [ServiceControl Management](/servicecontrol/license.md#servicecontrol-management-app) tool, set up ServiceControl to monitor endpoints using SQL Server transport.
+	 
+	* Use `Particular.ServiceControl.SQL` as the instance name
+	* Use local `ServiceControl` on local SQL Server Express as the database (ServiceControl Manager will automatically create queue tables in the database)  
+ 1. Ensure the `ServiceControl` process is running before running the sample.
+ 1. In the same instance of SQL Server, create databases for the endpoints: `sales`, `shipping` and `adapter`  
+ 1. [Install ServicePulse](https://docs.particular.net/servicepulse/installation)
+
+NOTE: In order to use a different SQL Server instance than local SQL Server Express, make sure the connection strings are updated in all places.
```

Comment:
We could simplify it:

NOTE: In order to connect to a different SQL Server instance, ensure all database connection strings are updated in the sample.